app/code/Magento/Sales/Model/Order.php

```diff
@@ -1946,7 +1946,7 @@ public function addRelatedObject(\Magento\Framework\Model\AbstractModel $object)
     /**
      * Get formatted order created date in store timezone
      *
-     * @param string $format date format type (short|medium|long|full)
+     * @param int $format date format type (0 = short|1 = medium|2 = long|3 = full)
```

Comment:
Instead of hardcoding numbers add note that constants like `\IntlDateFormatter::SHORT` must be used.

Also, please squash changes into a single commit.

Reply:
@orlangur , Updated code as per suggestion. Please review.
Thanks

Reply:
@jaimin-ktpl please squash changes into a single commit and we are good to go.

Reply:
Hello @orlangur , I have squashed changes into one commit.